clusterdock/cli.py

```diff
@@ -124,13 +124,10 @@ def main():
                                help="Don't actually perform manage actions")
     manage_subparsers = manage_parser.add_subparsers(dest='manage_action')
     manage_subparsers.required = True
-    manage_subparsers.add_parser('nuke')
 
-    remove_parser = manage_subparsers.add_parser('remove')
-    remove_parser.add_argument('-n', '--network',
-                               help='Network to remove (including all attached containers)',
-                               metavar='ntwrk',
-                               nargs='+')
+    nuke_parser = manage_subparsers.add_parser('nuke')
+    nuke_parser.add_argument('cluster_name', nargs='?',
```

Comment:
I think we should stick with using `remove`. `nuke` is `nuke` because it destroys everything (like a nuke); if we want to be more focused, we shouldn't use `nuke`.